wcvendors/wcvendors
# New Admin Vendor Manager Page

### All Submissions:






### Changes proposed in this Pull Request:



Closes #629 .

### How to test the changes in this Pull Request:

1. Login as Admin account
2. See submenu Vendors on WC Vendors menu.

### Other information:
